ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,55 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk=None, partitions=None, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :param partitions: Partitions DAL object list
+        :return: None
+        """
+        if disk is None and partitions is None:
+            return
+        if disk is None:
+            disk = partitions[0].disk
```

Comment:
If partitions provided is an empty list, this will throw an index out of range error